aidial_adapter_bedrock/llm/model/adapter.py

```diff
@@ -76,9 +77,26 @@ async def get_bedrock_adapter(
             ChatCompletionDeployment.STABILITY_STABLE_DIFFUSION_XL
             | ChatCompletionDeployment.STABILITY_STABLE_DIFFUSION_XL_V1
         ):
-            return StabilityAdapter.create(
+            return StabilityV1Adapter.create(
                 await Bedrock.acreate(aws_client_config), model, api_key
             )
+        case (
+            ChatCompletionDeployment.STABILITY_STABLE_IMAGE_CORE_V1
+            | ChatCompletionDeployment.STABILITY_STABLE_IMAGE_ULTRA_V1
+        ):
+            return StabilityV2Adapter.create(
```

Comment:
Stability actually name this API `StabilityAI REST API (v2beta)`, so I changed the naming from previous commits